### PR TITLE
Completed html-relative-links

### PR DESCRIPTION
**Pull request by m-Cho1**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Minji, you should not be using root URLs for this, please complete this again making use of `..`

**Comment by m-Cho1** [comment]
image

**Comment by m-Cho1** [comment]
image

File: html-relative-links/about-me/about-me.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
```
Comment (codingcodymiller) [inline html-relative-links/about-me/about-me.html:14]: In this case, you are exiting the current directory just to navigate right back into it, that is unnecessary. Instead, you can simply state the name of the file you want to show.

File: html-relative-links/contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
```
Comment (codingcodymiller) [inline html-relative-links/contact-us/contact-us.html:16]: In this case, you are exiting the current directory just to navigate right back into it, that is unnecessary. Instead, you can simply state the name of the file you want to show.

File: html-relative-links/projects/projects.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
```
Comment (codingcodymiller) [inline html-relative-links/projects/projects.html:15]: In this case, you are exiting the current directory just to navigate right back into it, that is unnecessary. Instead, you can simply state the name of the file you want to show.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by m-Cho1** [comment]
edited! 
image

**Pull request merged**